mapper/mapper.go

```diff
@@ -287,12 +287,16 @@ func CreateReleaseCalendar(basePage coreModel.Page, params queryparams.Validated
 	if totalResults > response.Breakdown.Total {
 		totalResults = response.Breakdown.Total
 	}
+	var currentPage int = queryparams.CalculatePageNumber(params.Offset, params.Limit)
+	var itemsPerPage int = params.Limit
```

Comment:
`itemsPerPage` is definitely a more descriptive alias for `params.Limit` - now it exists, it might as well be moved up before line 290 and used to replace all instanced of `params.Limit`.

Reply:
this is done now.